app/views/suggestions/_suggestion.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if @suggestions.class == Array %>
+  <% @suggestions.each do |hash|%>
+
+  <div class = "suggestion">
+    <% if hash.class == TunesTakeoutWrapper %>
```

Comment:
Is it possible for this to be something other than TunesTakeoutWrapper? If so, should there be an "else" clause?